src/components/Button/Button.style.ts

```diff
@@ -1,56 +1,5 @@
 import styled from 'styled-components';
 import Button from '@material-ui/core/Button';
-import { COLOR_MAP } from 'common/colors';
-
-const disableRippleProps = {
-  disableRipple: true,
-  disableElevation: true,
-  disableFocusRipple: true,
-};
-
-/**
- * Button that looks like a link.
- */
```

Comment:
These buttons are not used anywhere - The BaseButton here should probably be removed in favor of the design system buttons.